src/BuildScriptGenerator/DotNetCore/DotnetCoreLanguageDetector.cs

```diff
@@ -20,17 +20,20 @@ internal class DotNetCoreLanguageDetector : ILanguageDetector
         private readonly DotNetCoreScriptGeneratorOptions _scriptGeneratorOptions;
         DefaultProjectFileProvider _projectFileProvider;
         private readonly ILogger<DotNetCoreLanguageDetector> _logger;
+        private readonly IWriter _writer;
```

Comment:
don't we need this for other platforms as well? what happens if it's a node project and we run oryx build in root?

Reply:
There won't be any change in behavior across the other platforms since they don't currently need to write anything to standard output, but I went ahead and added the writer to the other platform language detector classes so they're able to use it if needed in the future.